.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Bug Report
+about: Report a bug encountered while using Nessie
+labels: kind/bug
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible.-->
+
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Details**:
+- Nessie server version and type (docker/built from source):
+- Nessie client version (pynessie):
```

Comment:
a bit more info about client here? It could be pynessie, UI, spark, delta etc. etc.
Maybe something like:

* client type
* client version

Reply:
done